Directory.Build.props

```diff
@@ -37,6 +37,6 @@
 
   <ItemGroup>
     <PackageReference Include="Nerdbank.GitVersioning" Version="3.3.37" PrivateAssets="All" />
-    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" PrivateAssets="All" Condition="'$(StyleCopEnabled)'=='true'" />
+    <PackageReference Include="StyleCop.Analyzers" Version="1.2.0-beta.556" PrivateAssets="All" Condition="'$(StyleCopEnabled)'=='true'" />
```

Comment:
I think we will need to use a release version here, not a beta version so that the final package doesn't have to be marked prerelease. Is there any particular reason to use a beta version here?

Reply:
There hasn't been a non-stable version for years now, and only this version understands recent C# 8+ with sensible formatting. I don't know why they won't commit to a new stable release, but they haven't. I use this beta version in all my own and work protects.

Reply:
Also, this package is only a build time dependency, it should not be a dependency of the shipped package.

Reply:
Yeah sounds good then.